### DISCUSSION TITLE
Add CBR format in the list of available convertions

### DISCUSSION THREAD

**Discussion by BilayKhan** [zelon88/HRConvert2#68] [Ideas]
Hello,
First thanks for this amazing job with HRConvert2!
I wonder if it would make sense to add the CBR/CBZ file format in conversion from/to PDF or images... Those are format for comics/manga.
At least I would have a real use of such feature.

**Comment by zelon88** [comment]
So I looked into it and CBR / CBZ files are just .zip containers full of .jpeg files.

That means it's pretty straightforward to have HRConvert2 process them as archives. I just finished adding support to v3.3.6 for CBR and CBZ files.

I found that 7z works better than zip for extracting them. So the process, currently, for working with CBR or CBZ files would be to upload them then convert them to an actual archive like zip, rar, or 7z. Then you will find that archive is full of jpeg files. Upload can then upload those and convert them to whatever you want.

I realize that this is less than ideal, as it requires some thought and effort on the users part. We could develop a convertComics() function to directly convert the input archive to an output archive with less steps, probably improving quality in the process. I'll add this to the list of things to do. Maybe if we gain some traction with comic fans it will help me get that done faster. :)

**Comment by zelon88** [comment]
https://github.com/zelon88/HRConvert2/commit/bc4d5dcb2f44f132d8e1b0788601e57d47f0cc2c

**Comment by zelon88** [comment]
I want to reopen and revist this to evaluate if it can / should be done natively in the core instead of requiring the user to upload two batches of files.

**Comment by zelon88** [comment]
Closing. This is now supported by multiple pipelines.
